scripts/meetingbaas.py

```
@@ -295,43 +294,56 @@ async def main(
+
+    # Wrap pipeline steps for logging
+    class LoggingStep:
+        def __init__(self, step, name):
+            self.step = step
+            self.name = name
+        async def __call__(self, *args, **kwargs):
+            result = await self.step(*args, **kwargs)
+            log_pipeline_step(self.name, result)
+            return result
+
+    wrapped_stt = LoggingStep(stt, "STT")
```

Ruff (0.11.9)

Local variable `wrapped_stt` is assigned to but never used

Remove assignment to unused variable `wrapped_stt`

(F841)